src/CommitmentsV2/SFA.DAS.CommitmentsV2/Validators/DraftApprenticeshipDetailsValidator.cs

```diff
@@ -0,0 +1,159 @@
+﻿using FluentValidation;
+using FluentValidation.Validators;
+using SFA.DAS.CommitmentsV2.Domain.Entities;
+using SFA.DAS.CommitmentsV2.Domain.Extensions;
+using SFA.DAS.CommitmentsV2.Domain.Interfaces;
+using SFA.DAS.CommitmentsV2.Domain.ValueObjects;
+using TrainingProgrammeStatus = SFA.DAS.Apprenticeships.Api.Types.TrainingProgrammeStatus;
+
+namespace SFA.DAS.CommitmentsV2.Validators
+{
+    public class DraftApprenticeshipDetailsValidator : AbstractValidator<DraftApprenticeshipDetails>
+    {
+        private class StartDateValidator : AbstractValidator<DraftApprenticeshipDetails>
+        {
+            public StartDateValidator(IAcademicYearDateProvider academicYearDateProvider)
+            {
+                RuleFor(ctx => ctx.StartDate)
```

Comment:
It looks as though the property names come back prefixed with the name of the class (eg DraftApprenticeshipDetails.StartDate) which means the user gets an error state but no errors show up in the UI.

Reply:
Fixed.